lite/tests/unittest_py/op/test_assign_op.py

```diff
@@ -13,28 +13,44 @@
 # limitations under the License.
 
 import sys
-sys.path.append('../../common')
-sys.path.append('../../../')
+sys.path.append('../')
 
-import test_assign_op_base
 from auto_scan_test import AutoScanTest, IgnoreReasons
 from program_config import TensorConfig, ProgramConfig, OpConfig, CxxConfig, TargetType, PrecisionType, DataLayoutType, Place
 import unittest
 
 import hypothesis
 from hypothesis import given, settings, seed, example, assume
-
+import hypothesis.strategies as st
+import argparse
 
 class TestAssignOp(AutoScanTest):
     def is_program_valid(self, program_config: ProgramConfig) -> bool:
         return True
 
     def sample_program_configs(self, draw):
-        return test_assign_op_base.sample_program_configs(draw)
+        in_shape = draw(st.lists(st.integers(min_value=1, max_value=8), max_size=2))
+        assign_op = OpConfig(
+            type = "assign",
+            inputs = {"X" : ["input_data"]},
+            outputs = {"Out": ["output_data"]},
+            attrs = {})
+        program_config = ProgramConfig(
+            ops=[assign_op],
+            weights={},
+            inputs={
+                "input_data":
+                TensorConfig(shape=in_shape)
+            },
+            outputs=["output_data"])
+        return program_config
 
     def sample_predictor_configs(self):
         config = CxxConfig()
-        config.set_valid_places({Place(TargetType.Host, PrecisionType.FP32, DataLayoutType.NCHW)})
+        if self.args.target == "host":
```

Comment:
The map from target to CxxConfig should be a shared method in the underlying base class.
Each test shouldn't implement it for each, duplicate code and easy to be buggy.

e.g.

```python
class BaseClassXXX:
  def get_predictor_configs(self) -> List[CxxConfig]: return self.target_to_predictor_configs(self.get_target())

  @staticmethod
  def target_to_predictor_configs(target:str) -> List[CxxConfig]: pass

  def get_target(self) -> str:
    return self.args.target



class Place:
  def __init__(self, target, precision, layout):
    pass

  def __str__(self): pass
```


The global schedule script:

1. Get all the place candidates

```bash
places=python << EOD
from autoscan import*

for place in product(target.all, precision.all, layout.all):
  print(place)
EOD
```

2. for each place in places, do the test
```
python xxx_test.py --place=place
```